docs/zh/latest/config.json

```diff
@@ -56,9 +56,11 @@
             "plugins/gzip",
             "plugins/real-ip",
             "plugins/server-info",
-            "plugins/ext-plugin-post-req",
             "plugins/ext-plugin-pre-req",
-            "plugins/ext-plugin-post-resp"
+            "plugins/ext-plugin-post-req",
+            "plugins/ext-plugin-post-resp",
+            "plugins/inspect",
```

Comment:
There is no inspect in the Chinese doc.

Reply:
@spacewander it should be good now. Thanks! 😄